Cargo.toml

```diff
@@ -24,13 +24,14 @@ diesel = { version = "1.4.5", features = ["chrono", "postgres", "r2d2"] }
 jsonwebtoken = "7.2.0"
 rand = "0.7.3"
 reqwest = { version = "0.11.2", features = ["blocking", "json"] }
-rocket = "0.4.5"
-rocket_contrib = {version = "0.4.5", features = ["json"]}
+rocket = "0.4.10"
```

Comment:
**n.b.** Latest Rust Nightly is not compatible with `rocket = 0.4.5`.

Reply:
Oh, dear, what did they do this time! :)